solr/solr-ref-guide/src/other-parsers.adoc

```diff
@@ -237,7 +237,7 @@ Comma separated list of tags for excluding queries from parameters above. See ex
 {!bool filter=foo should=bar}
 ----
 
-Parameters might also be multivalue references. The former example above is equivlent to 
```

Comment:
This and next empty space seems not needed. But 'equivlent' spelling could be fixed.

Reply:
okay, fixed the typo, but left spacing to match the other spacing.